### PR TITLE
JT-12 [getting participants from backend]

### PR DESCRIPTION
**Pull request by fstk1337**

### CODE REVIEW

File: vue-app/src/components/footer/Footer.vue [hunk 1]
```diff
@@ -15,47 +15,39 @@
...
         }
+    },
+    mounted() {
+      getContributors(this)
     }
 }
 
+function getContributors(obj) {
+    const axiosInstance = axios.create({
+      baseURL: 'http://localhost:8080',
```
Comment (pelmegov) [inline vue-app/src/components/footer/Footer.vue:35]: Let's extract it to configuration file.

File: vue-app/src/components/footer/Footer.vue [hunk 2]
```diff
@@ -15,47 +15,39 @@
...
-          ]
+          contributors: []
         }
+    },
+    mounted() {
+      getContributors(this)
     }
 }
 
+function getContributors(obj) {
```
Comment (pelmegov) [inline vue-app/src/components/footer/Footer.vue:33]: Vue has a methods section, we can use it to put our methods there without passing the obj reference.

File: stock-system/src/main/java/saffchen/controller/ParticipantController.java [hunk 3]
```diff
@@ -16,7 +17,8 @@ public class ParticipantController {
     private final ParticipantService participantService;
 
     @GetMapping
-    public List<ParticipantDto> getParticipants() {
+    public List<ParticipantDto> getParticipants(HttpServletResponse response) {
+        response.addHeader("Access-Control-Allow-Origin", "*");
```
Comment (pelmegov) [inline stock-system/src/main/java/saffchen/controller/ParticipantController.java:21]: It's better to separate business logic and infrastructure settings, let's try to find another and probably better way to allow origin.

**Review by pelmegov — CHANGES_REQUESTED** [review]

File: vue-app/src/components/footer/Footer.vue (hunk 1 above)
Comment (fstk1337) [reply vue-app/src/components/footer/Footer.vue]: Done.

File: vue-app/src/components/footer/Footer.vue (hunk 2 above)
Comment (fstk1337) [reply vue-app/src/components/footer/Footer.vue]: Done.

File: stock-system/src/main/java/saffchen/controller/ParticipantController.java (hunk 3 above)
Comment (fstk1337) [reply stock-system/src/main/java/saffchen/controller/ParticipantController.java]: Done.

**Review by saffchen — APPROVED** [review]

File: stock-system/src/main/java/saffchen/controller/ParticipantController.java [hunk 4]
```diff
@@ -16,7 +17,7 @@ public class ParticipantController {
     private final ParticipantService participantService;
 
     @GetMapping
-    public List<ParticipantDto> getParticipants() {
+    public List<ParticipantDto> getParticipants(HttpServletResponse response) {
```
Comment (pelmegov) [inline stock-system/src/main/java/saffchen/controller/ParticipantController.java:20]: Let's remove unnecessary parameter.

**Review by pelmegov — APPROVED** [review]

File: stock-system/src/main/java/saffchen/controller/ParticipantController.java (hunk 4 above)
Comment (fstk1337) [reply stock-system/src/main/java/saffchen/controller/ParticipantController.java]: Done.

**Pull request merged**